client/src/test/java/org/spine3/base/IdentifiersShould.java

```diff
@@ -59,7 +59,7 @@ public void have_private_constructor() {
     @Test(expected = IllegalArgumentException.class)
     public void throw_exception_when_object_of_unsupported_class_passed() {
         //noinspection UnnecessaryBoxing
```

Comment:
Let's use an annotation-based suppression instead. Even though it's in the test code.